modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -257,6 +260,20 @@ When forwarding logs to external services, additional metadata might be availabl
 | labels_type | Specifies the type of deployment that the pod is running. | MuleApplication
 |===
 
+== Send a Test Message
+
+You can send a test message to verify connectivity between Runtime Fabric and the third-party log forwarding solution:
+
+. In Runtime Manager, select *Runtime Fabrics* in the sidebar navigation.
+. Select the applicable Runtime Fabric based on the name used during installation.
+. Select the *Log Forwarding* tab.
+. Select *Forward application logs to an external service*.
+. Select a service and configure the fields as needed for your log forwarding solution.
+. Save the changes and apply the configuration to the Runtime Fabric cluster.
+. To test the configuration and reachability of the log forwarding solution:
```

Comment:
What is "reachability"? Do we mean to test the configuration and to verify that {something} sends {something} to the log forwarding solution?